From 60857229aab13ccc426572a43ab891409bb76ea4 Mon Sep 17 00:00:00 2001
From: Khem Raj <raj.khem@gmail.com>
Date: Sun, 9 Sep 2018 22:52:55 -0700
Subject: [PATCH] Add missing multi-line separator

Otherwise this fails to build ( found on mips )

Fixes
contrib/modules/surface_matching/src/hash_murmur86.hpp:97:15: error:
expected constructor, destructor, or type conversion before '(' token
     && defined(__GNUC__) && (__GNUC__>4 || (__GNUC__==4 &&
__GNUC_MINOR__>=3))
               ^

Upstream-Status: Submitted [https://github.com/opencv/opencv_contrib/pull/1764]
Signed-off-by: Khem Raj <raj.khem@gmail.com>
---
 modules/surface_matching/src/hash_murmur86.hpp | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/modules/surface_matching/src/hash_murmur86.hpp b/modules/surface_matching/src/hash_murmur86.hpp
index 1edf6bf4..0477d37e 100644
--- a/modules/surface_matching/src/hash_murmur86.hpp
+++ b/modules/surface_matching/src/hash_murmur86.hpp
@@ -93,7 +93,7 @@ void hashMurmurx86 ( const void * key, const int len, const uint seed, void * ou
 /* Now find best way we can to READ_UINT32 */
 #ifndef WORDS_BIGENDIAN
 # define READ_UINT32(ptr)   (*((uint32_t*)(ptr)))
-#elif defined(WORDS_BIGENDIAN)
+#elif defined(WORDS_BIGENDIAN) \
     && defined(__GNUC__) && (__GNUC__>4 || (__GNUC__==4 && __GNUC_MINOR__>=3))
 # define READ_UINT32(ptr)   (__builtin_bswap32(*((uint32_t*)(ptr))))
 #endif
-- 
2.18.0

